### PR TITLE
fix(emr): solve emr_cluster_publicly_accesible error

### PR DESCRIPTION
**Pull request by MrCloudSec**
### Description

Solve the following error in `emr_cluster_publicly_accesible` check:
`emr_cluster_publicly_accesible -- AttributeError[36]: 'NoneType' object has no attribute 'append'`

### License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: prowler/providers/aws/services/emr/emr_cluster_publicly_accesible/emr_cluster_publicly_accesible.py [hunk 1]
```diff
@@ -32,7 +32,10 @@ def execute(self):
                     master_node_sg_groups = deepcopy(
                         cluster.master.additional_security_groups_id
                     )
-                    master_node_sg_groups.append(cluster.master.security_group_id)
+                    if master_node_sg_groups:
```
Comment (jfagoagas) [inline prowler/providers/aws/services/emr/emr_cluster_publicly_accesible/emr_cluster_publicly_accesible.py:35]: Do the same for the `slave_node_sg_groups` please.

**Review by jfagoagas — CHANGES_REQUESTED** [review]

File: prowler/providers/aws/services/emr/emr_cluster_publicly_accesible/emr_cluster_publicly_accesible.py (hunk 1 above)
Comment (MrCloudSec) [reply prowler/providers/aws/services/emr/emr_cluster_publicly_accesible/emr_cluster_publicly_accesible.py:35]: Done

File: prowler/providers/aws/services/emr/emr_cluster_publicly_accesible/emr_cluster_publicly_accesible.py
Comment (jfagoagas) [inline prowler/providers/aws/services/emr/emr_cluster_publicly_accesible/emr_cluster_publicly_accesible.py:1]: Could you create some test to check the behaviour when the following returns `None`?
```
master_node_sg_groups = deepcopy(
                        cluster.master.additional_security_groups_id
                    )
```

**Review by jfagoagas — CHANGES_REQUESTED** [review]

File: prowler/providers/aws/services/emr/emr_cluster_publicly_accesible/emr_cluster_publicly_accesible.py
Comment (MrCloudSec) [reply prowler/providers/aws/services/emr/emr_cluster_publicly_accesible/emr_cluster_publicly_accesible.py:1]: Done

**Review by jfagoagas — APPROVED** [review]

**Pull request merged**